### ISSUE TITLE
Disable Kernel symbol unavailable log

### ISSUE THREAD

**Issue by AdamKaczmarski** [pyroscope-io/pyroscope-java#85]
Hi,
Is it possible to disable / hide this log from the agent?
```
[WARN] Kernel symbols are unavailable due to restrictions. Try
  sysctl kernel.kptr_restrict=0
  sysctl kernel.perf_event_paranoid=1
```

I am using the tracing integration with opentel.
https://pyroscope.io/docs/java-tracing/

It's probably coming from here.
https://github.com/async-profiler/async-profiler#basic-usage

**Comment by korniltsev** [comment]
AP has loglevel argument, I will need need to pass it from pyroscope to AP.

In the meantime I suppose you can switch to itimer instead of cpu

**Comment by korniltsev** [comment]
fixed in #91 

You can suppress it with `PYROSCOPE_AP_LOG_LEVEL="none"` or `PYROSCOPE_AP_EXTRA_ARGUMENTS="loglevel=none"` or `PYROSCOPE_AP_EXTRA_ARGUMENTS="alluser"`
